README.md

```diff
@@ -281,14 +281,30 @@ Description of arguments for these scripts are as follows. See examples below.
 * instance: Cloud Sql Instance
 
 ###Examples:
+#### Generate denormalized tables in the BigQuery cdr only one time when it is released or as needed
+`./project.rb make-bq-denormalized-tables --bq-project all-of-us-ehr-dev --bq-dataset test_merge_dec26 `
+##### Result is
+1. The BigQuery dataset has new denormalized tables for cohort builder to work.
 #### Generate count data in BigQuery from a cdr release
 `./project.rb generate-cdr-counts --bq-project all-of-us-ehr-dev --bq-dataset test_merge_dec26 --workbench-project all-of-us-workbench-test --public-project all-of-us-workbench-test --cdr-version 20180206 --bin-size 20 --bucket all-of-us-workbench-private-cloudsql`
 ##### Result is
 1. BigQuery datasets:  all-of-us-workbench-test:cdr20180206 and all-of-us-workbench-test:public20180206
-2. CSV dumps of tables in bucket all-of-us-workbench-private-cloudsql: cdr20180206/*.csv and public20180206/*.csv with public counts in multiples of bin-size
+2. CSV dumps of tables in bucket all-of-us-workbench-private-cloudsql: cdr20180206/*.csv.gz and public20180206/*.csv.gz with public counts in multiples of bin-size
 3. Browse csvs in browser like here :https://console.cloud.google.com/storage/browser?project=all-of-us-workbench-test&organizationId=394551486437
 3. Note cdr-version can be ''  to make datasets named cdr and public
-#### Generate local mysql databases -- cdr and public for data generated above
+#### Generate cloudsql databases from a bucket without downloading the data
+`# Once for private cdr`
+
+`./project.rb generate-cloudsql-db --project all-of-us-workbench-test --instance workbenchmaindb --database cdr20180913 --bucket all-of-us-workbench-private-cloudsql/cdr20180913`
+
+`# Once for public cdr`
+
+`./project.rb generate-cloudsql-db --project all-of-us-workbench-test --instance workbenchmaindb --database public20180913 --bucket all-of-us-workbench-private-cloudsql/public20180913`
+##### Result is
+1. Databases are live on cloudsql.
+2. TODO - Run .project.rb update-cdr-versions to use them in your environment ??
+
+#### Generate local mysql databases -- cdr and public for data generated above if you need to develop with a full test database
```

Comment:
Data generated above seems to be lil confusing. May be we can change it to "from data in csvs in buckets" or whatever.